templatescompiler/erbrenderer/erb_renderer.rb

```diff
@@ -169,22 +166,25 @@ def dump(*args)
 end
 
 class ERBRenderer
-  def initialize(context)
-    @context = context
+  def initialize(json_context_path)
+    @json_context_path = json_context_path
   end
 
   def render(src_path, dst_path)
-    erb = ERB.new(File.read(src_path), safe_level = nil, trim_mode = "-")
```

Comment:
nit: while i dont feel strongly about it (and frankly it doesnt matter since its out of process from the cli) in theory its nice to call out that its running in the same process...